.github/workflows/publish-documentation.yml

```diff
@@ -18,9 +18,7 @@ jobs:
       - name: Execution permission for Gradle wrapper
         run: chmod +x gradlew
       - name: Dokka
-        run: ./gradlew dokka
-      - name: Gather modules documentation
-        run: ./gradlew gatherDokkaDocumentation
+        run: ./gradlew dokkaHtmlMultiModule
       - name: Publish GitHub pages
```

Comment:
I haven't seen the generated output yet, but I'm assuming you might have to modify the folder in this step to `build/dokka/html` judging by the other changes.

Reply:
Yes, I had to, didn't see this one hahahahhaha